src/LexerParser.ts

```diff
@@ -26,6 +27,9 @@ export function getLexerParserFn(
     let memoizedStatements = new Map<string, Promise<Stmt.Statement[]>>();
     return async function parse(filenames: string[]): Promise<Stmt.Statement[]> {
         async function lexAndParseFile(filename: string) {
+            if (process.platform === "win32") {
+                filename = filename.split(path.sep).join(path.posix.sep);
+            }
```

Comment:
How about this instead? I think it's more efficient, and a little easier to read. This also makes it a little more generic, forcing everything to forward slash regardless of platform. I think it makes sense to use this pattern in all of the changed path locations in this PR

```suggestion
            filename = filename.replace(/[\/\\]+/g, path.posix.sep);
```

Reply:
Done